frontend/src/patientApp/timeOfTest/DOB.tsx

```diff
@@ -15,8 +15,18 @@ import { dateFromStrings } from "../../app/utils/date";
 const DOB = () => {
   const { t } = useTranslation();
 
-  const dispatch = useDispatch();
   const plid = useSelector((state: any) => state.plid);
+
+  useEffect(() => {
+    PxpApi.getObfuscatedPatientName(plid)
```

Comment:
Since this API call is happening after the initial render, I think it might be good to use a `loading` state and render the `LoadingCard` component while the API call is happening, so that we don't show `'s date of birth` in the UI

Reply:
Love it 🙌 added in e6a3261

Blink and you'll miss it:

https://user-images.githubusercontent.com/27730981/152205377-4d6ff47b-750d-4ccd-b2a9-53f42c54f2b3.mov